[2025-08-18T17:06:49.170+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-18T17:06:49.220+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.validate_transform manual__2025-08-18T17:02:23.468906+00:00 [queued]>
[2025-08-18T17:06:49.234+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.validate_transform manual__2025-08-18T17:02:23.468906+00:00 [queued]>
[2025-08-18T17:06:49.234+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2025-08-18T17:06:49.264+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): validate_transform> on 2025-08-18 17:02:23.468906+00:00
[2025-08-18T17:06:49.272+0000] {standard_task_runner.py:63} INFO - Started process 8048 to run task
[2025-08-18T17:06:49.277+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_batch_data', 'validate_transform', 'manual__2025-08-18T17:02:23.468906+00:00', '--job-id', '407', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmp5f9ljaqn']
[2025-08-18T17:06:49.280+0000] {standard_task_runner.py:91} INFO - Job 407: Subtask validate_transform
[2025-08-18T17:06:49.335+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_batch_data.validate_transform manual__2025-08-18T17:02:23.468906+00:00 [running]> on host 47205959681c
[2025-08-18T17:06:49.449+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_batch_data' AIRFLOW_CTX_TASK_ID='validate_transform' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T17:02:23.468906+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T17:02:23.468906+00:00'
[2025-08-18T17:06:49.450+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-18T17:06:49.468+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T17:06:49.468+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        ROW_COUNT=$(docker exec namenode hdfs dfs -count /storage/hdfs/processed/golden_dataset | awk \'{print $2}\')\n        if [ "$ROW_COUNT" -lt 1 ]; then\n            echo "Transform failed: no rows in golden dataset" && exit 1\n        else\n            echo "Transform OK ($ROW_COUNT rows)"\n        fi\n\n        ']
[2025-08-18T17:06:49.481+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T17:06:51.670+0000] {subprocess.py:93} INFO - Transform OK (112 rows)
[2025-08-18T17:06:51.671+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-18T17:06:51.672+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-18T17:06:51.708+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=youtube_pipeline_batch_data, task_id=validate_transform, execution_date=20250818T170223, start_date=20250818T170649, end_date=20250818T170651
[2025-08-18T17:06:51.748+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-18T17:06:51.788+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-18T17:06:51.797+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
